code-challenges-05/linked-list.js

```diff
@@ -0,0 +1,90 @@
+//testing 123
+// class Node {                                                                                        
+//   constructor(data){
+//     this.data = data;
+//     this.next = this.head;
```

Comment:
This implementation is very close. When you make a new Node, its next property shoould probably be set to null (not to this.head). Try `this.next = null`!